test/get_previous_releases.py

```diff
@@ -104,7 +104,11 @@ def download_binary(tag, args) -> int:
     tarballHash = hasher.hexdigest()
 
     if tarballHash not in SHA256_SUMS or SHA256_SUMS[tarballHash] != tarball:
-        print("Checksum did not match")
+        if checksum_exists(tag[1:]):
```

Comment:
nit: could also pass the full filename (`tarball`) here, then the function `checksum_exists` below wouldn't need to add the "bitcoin-" prefix and is more specific?

Reply:
I purposely avoided passing the full name in case the naming convention changes it will still work. Thinking more about it would be helpful to check the full name as it will throw the checksum doesn't exist if we are missing a specific OS/arch on the list even if we included some for that same version.